Packs/FeedRecordedFuture/ReleaseNotes/1_0_31.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Recorded Future RiskList Feed
+- Updated the integration description to include a notice advising users against fetching 100,000 or more indicators per instance.
```

Comment:
You also changed the default value for expiration, lets add that.

Reply:
Nice catch!